js&css/extension/www.youtube.com/styles.css

```diff
@@ -476,7 +476,7 @@ ytd-guide-section-renderer .it-button::after {
 	align-items: center;
 	content: 'Blocklist';
 	text-transform: uppercase;
-	color: var(--yt-spec-icon-active-other, #fff);
```

Comment:
(maybe the variable is once was a reasoned choice,  so one _could_ modify the color's value or replace it with another variable?)